src/app/components/UserAssets/index.tsx

```diff
@@ -236,7 +236,7 @@ function AssetRow({
   const dollarValueMynt = useDollarValueMynt(tokens);
   const _dollarValue = useMemo(
```

Comment:
Rename it to something else, `_dollarValue` is really lazy as you have a variable named `dollarValue` two lines above it. I would say it could be `assetDollarValue`, what do you think @soulBit ?

Reply:
tiltom's suggestion sounds good here 👍🏻